src/components/RangeSlider/index.scss

```diff
@@ -174,6 +168,59 @@ $lower-background: linear-gradient(to top, $lower-color, $lower-color) 100% 50%
   }
 }
 
-.sliderErrorContainer {
-  margin: -10px 0 10px 0;
+
+.sliderGridContainer {
+  width: 100%;
+  position: relative;
+  display: grid;
```

Comment:
When thinking about it, I think this should actually be achieved using flexbox

Reply:
Then you also need less feature-queries

Reply:
done